mobile/lib/shared/models/user.dart

```diff
@@ -18,11 +18,12 @@ class User {
     this.isPartnerSharedWith = false,
     this.profileImagePath = '',
     this.memoryEnabled = true,
+    this.inTimeline = false,
   });
 
   Id get isarId => fastHash(id);
 
-  User.fromDto(UserResponseDto dto)
+  User.fromUserResponseDto(UserResponseDto dto)
```

Comment:
@fyfrey I modify the method call to make a distinction when parsing the `UserResponseDto` and `PartnerResponseDto` to make a User in the mobile app

Reply:
distinguishing those two cases is good! can we make it `fromUserDto`and `fromPartnerDto` - no need for `Response` :)